var/spack/repos/builtin/packages/libquo/package.py

```diff
@@ -28,13 +28,22 @@ class Libquo(AutotoolsPackage):
     depends_on('automake', when='@develop', type='build')
     depends_on('libtool',  when='@develop', type='build')
 
+    variant('force-pic', default=False,
```

Comment:
Let's name the variant `pic` to match other packages.

Reply:
I originally named the variant `pic`, but the package maintainer pointed out that this was possibly misleading because the package by default already produces PIC code. The underlying problem is the interaction between PGI and `libtool` or as he states it:

> may I suggest a small name and description string change? After thinking about it more, perhaps something like  
> --force-pic (and also clearly reflecting this in the help) might be more informative, less confusing. Quo already 
> does by default produce PIC, it is just that sometimes libtool doesn't get it right (e.g., in your case with PGI). Just 
> something to consider. Thank you.

I wonder now ... Instead of making this a variant, would it be better to enforce this in the recipe? For example:
``
 if '%pgi' in self.spec:
            config_args.append('CFLAGS={0}'.format(self.compiler.cc_pic_flag))
            config_args.append('FCFLAGS={0}'.format(self.compiler.fc_pic_flag))
``

Reply:
> ```
>  if '%pgi' in self.spec:
>             config_args.append('CFLAGS={0}'.format(self.compiler.cc_pic_flag))
>             config_args.append('FCFLAGS={0}'.format(self.compiler.fc_pic_flag))
> ```

That seems to work!  @samuelkgutierrez, would you be alright with this rather than a variant?